src/services/userService.js

```diff
@@ -44,10 +47,34 @@ const getUserInfo = async (email) => {
   return userInfo;
 };
 
+const saveUserAvatar = async (file, userId) => {
+  const pathName = file.path;
+  const fileExtension = file.originalname.split('.').pop();
+  const newAvatar = `${userId}.${fileExtension}`;

```

Comment:
I think in this case each file won't have unique name. It is better when each file has unique name